test_support/file_operations.go

```diff
@@ -0,0 +1,40 @@
+package test_support
```

Comment:
The file name should be `file_operations_test.go` so that the code is only available to tests. (If it's used outside tests, then it's package would need renaming.)

Reply:
If I rename it to "file_operations_test.go", I get complaints that there are no non-test .go files in that directory. My options are: 1) name it as-is, 2) move it to the config package alongside the tests that use it, 3) put some dummy .go file alongside it to appease the compiler, 4) something else that I'm unaware of that will make it happy. Any thoughts?